### PR TITLE
Nikolina/media hunt project

### PR DESCRIPTION
**Pull request by Ninolina88**
Done Us #55 and #56

### CODE REVIEW

File: Public/media-hunt/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,574 @@
+{
```
Comment (pvukmanovic-pannonians) [inline Public/media-hunt/package-lock.json:1]: Molim te obriši package-lock. Koristite yarn.

**Review by pvukmanovic-pannonians — CHANGES_REQUESTED** [review]

**Comment by pvukmanovic-pannonians** [comment]
Please remove this content from index.js. that content will come near the end of the project.

File: nikolina/package.json [hunk 2]
```diff
@@ -1,19 +0,0 @@
...
-    "start": "node index.js",
-    "inquirer": "yarn workspace nikolina-inquirer"
-  },
-  "keywords": [],
-  "author": "",
-  "license": "ISC",
-  "type": "commonjs",
-  "private":"true",
-  "workspaces": ["./inquirer"]
-}
```
[the remark this answers is not in the document]
Comment (Ninolina88) [reply nikolina/package.json]: igrala sam se kako ne treba na gitu. Reseno.

File: Public/media-hunt/package-lock.json (hunk 1 above)
Comment (Ninolina88) [reply Public/media-hunt/package-lock.json]: Obrisano.

**Comment by Ninolina88** [comment]
Content from index.js removed as per request.

**Review by pvukmanovic-pannonians — DISMISSED** [review]

File: nikolina/package.json [hunk 3]
```diff
@@ -4,16 +4,18 @@
...
   "author": "",
   "license": "ISC",
   "type": "commonjs",
-  "private":"true",
-  "workspaces": ["./inquirer"]
+  "private": true,
+  "workspaces": [
+    "./inquirer",
+    "./explorer"
+  ]
```
Comment (pvukmanovic-pannonians) [inline nikolina/package.json:20]: Please revert these changes. Please don't make any changes to files outside of the scope, and the scope was to scaffold project.

**Review by pvukmanovic-pannonians — CHANGES_REQUESTED** [review]
Please add to the root `package.json` file workspace so that media-hunt's scripts can be executed.

File: package.json [hunk 4]
```diff
@@ -2,7 +2,7 @@
   "name": "learning-js",
   "version": "1.0.0",
   "private": true,
-  "workspaces": ["petar", "aleksandra", "nikolina", "david"],
+  "workspaces": ["petar", "aleksandra", "nikolina", "david", "Public/media-hunt"],
   "description": "**Shortcuts and Most used commands in Linux**",
   "main": "index.js",
   "scripts": {
```
Comment (pvukmanovic-pannonians) [inline package.json:8]: Please also add the script so you could execute commands. Like `yarn nikolina appname`, but for this to be something like `yarn public moviehunt`

**Review by pvukmanovic-pannonians — COMMENTED** [review]
Let's finalize root package json

**Review by pvukmanovic-pannonians — CHANGES_REQUESTED** [review]
Missing bits for package json

**Review by pvukmanovic-pannonians — APPROVED** [review]
LGTM (looks good to me)

**Pull request merged**